public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx

```diff
@@ -59,16 +60,13 @@ export class PanelDataPane extends SceneObjectBase<PanelDataPaneState> {
   }
 
   private onActivate() {
-    const panel = this.panelManager.state.panel;
-    this.setupPanelSubscription(panel);
```

Comment:
we no longer need a panel sub as we hold everything we need on the panelVizManager. panel sub would also become pointless when switching to table view, thus deactivating the vizPanel and removing it's subs.